src/components/ListDetail/ListDetail.module.css

```diff
@@ -46,7 +46,7 @@
   @media all and (min-width: $eds-bp-lg) {
     flex: 1; /* 1 */
     padding: var(--eds-size-5) var(--eds-size-8);
-    box-shadow: -6px 0px 10px -2px var(--eds-theme-color-neutral-subtle-border); /* 1 */
+    box-shadow: -6px 0px 10px -2px var(--eds-theme-color-border-neutral-subtle); /* 1 */
```

Comment:
should we define this as a theme box shadow?

Reply:
Maybe a tech debt item should be to sweep through components to find outliers like this to discuss with design, i.e. should this really be a unique box shadow anyway?